mod_reforged/hooks/skills/racial/golem_racial.nut

```diff
@@ -1,4 +1,18 @@
 ::mods_hookExactClass("skills/racial/golem_racial", function(o) {
+	o.onAdded = function()
+	{
+		local base = this.m.BaseProperties;
+
+		base.IsAffectedByInjuries = false;
+		base.IsAffectedByNight = false;
+
```

Comment:
These empty lines to separate "AffectedBy" from "ImmuneTo" properties seem unnecessary to me.